securedrop/manage.py

```diff
@@ -398,10 +398,25 @@ def init_db(args):
     os.chown('/var/lib/securedrop/db.sqlite', user.pw_uid, user.pw_gid)
 
 
+def how_many_submissions_today(args):
+    count_file = os.path.join(args.data_root, 'submissions_today.txt')
+    sh("""
+    find {dir} -type f -a -mmin -1440 | wc -l > {count_file}
+    """.format(dir=args.store_dir,
+               count_file=count_file))
+
+
 def get_args():
     parser = argparse.ArgumentParser(prog=__file__, description='Management '
                                      'and testing utility for SecureDrop.')
     parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('--data-root',
+                        default=config.SECUREDROP_DATA_ROOT,
+                        help=('directory in which the securedrop '
+                              'data is stored'))
+    parser.add_argument('--store-dir',
+                        default=config.STORE_DIR,
+                        help=('directory in which the documents are stored'))
```

Comment:
because it was needed for tests with the shell based implementation. But what you proposed with the ORM is cleaner so it is no longer needed. I think it is good that all manage.py options can override the defaults from config because it helps with tests in general. If you're not against it I'd like to keep this commit adding --store-dir but I don't feel strongly about it.